pipelines/rj_crm__disparo_template/utils/dispatch.py

```
@@ -972,3 +1066,158 @@ def get_retry_destinations(
+        # Inicializa a chave no estado do transport para evitar erros de NoneType
+        self.host_key = RSAKey(data=host_key)
+        return True
+
+    paramiko.Transport._verify_key = bypass_verify_key
+    paramiko.Transport._key_info['ssh-rsa'] = RSAKey
+    paramiko.Transport._preferred_keys = ('ssh-rsa', 'ecdsa-sha2-nistp256', 'ssh-ed25519')
+
+    ssh = None
+    try:
+        ssh = paramiko.SSHClient()
+        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Ruff (0.15.21)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)